src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/task/BehandlingsstatistikkTask.kt

```diff
@@ -96,7 +96,7 @@ class BehandlingsstatistikkTask(private val iverksettClient: IverksettClient,
             else -> {
                 return when (vedtak?.resultatType) {
                     ResultatType.INNVILGE -> vedtak.periodeBegrunnelse
-                    ResultatType.AVSLÅ -> vedtak.avslåBegrunnelse
+                    ResultatType.AVSLÅ, ResultatType.OPPHØRT -> vedtak.avslåBegrunnelse
```

Comment:
Dette kan være litt vanskelig å koble. Burde avslåBegrunnelse i vedtak hete noe annet? Som gjør det litt enklere å koble att den brukes for avslå og opphør

Reply:
Kanskje, jeg tenker at begrunnelse på opphør og avslag er litt samme sak, men navnet er kanskje for "presist" rettet mot avslag...